tests/test_module_hamevo.py

```diff
@@ -222,3 +221,36 @@ def overlap(state1: torch.Tensor, state2: torch.Tensor) -> torch.Tensor:
     result = overlap(psi_star, psi)
     assert result.size() == (batch_size,)
     assert torch.allclose(result, target)
+
+
+def test_hamevo_endianness() -> None:
+    # m = torch.rand(2,2)
```

Comment:
Remove ?

Reply:
wanted to keep this comment, because its how I constructed the matrices initially, just in case we can to change things

Reply:
Ah, OK. Maybe add a comment then. Currently, it feels like left-over from debugging.

Reply:
actually, removed them. they really do look like debugging 😅